kodilla-testing/src/main/java/com/kodilla/testing/forum/statistics/StatisticsProcessor.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.testing.forum.statistics;
+
+public class StatisticsProcessor {
+
+    public StatisticsCollector calculateAdvStatistics(Statistics statistics) {
+
+        StatisticsCollector collector = new StatisticsCollector(0, 0, 0, 0, 0, 0);
```

Comment:
to niepotrzebne, Java sama zainicjalizuje wartości na zero. Dopisz poprostu bezparametrowy konstruktor.